lib/src/interfaces/Options.ts

```diff
@@ -3,7 +3,7 @@ import { ImageRequireSource, ImageSourcePropType, Insets } from 'react-native';
 
 // TODO: Import ColorValue instead when upgrading @types/react-native to 0.63+
 // Only assign PlatformColor or DynamicColorIOS as a Color symbol!
-declare type Color = string | symbol;
+export declare type RNNColor = string | symbol | ModedColour;
```

Comment:
I think `RNNColor` should be renamed to `Color` as it doesn't have much sense to add a prefix in js